src/main/java/frc/lib5k/utils/RobotLogger.java

```diff
@@ -161,7 +162,7 @@ private String toString(String component, String msg, Level log_level) {
             break;
         }
 
-        return String.format("%s%s %s", level_str, (component.equals("")) ? component : "[" + component + "]", msg);
+        return String.format("%s%s%s %s", level_str, FPGAClock.getFPGASeconds(),(component.equals("")) ? component : "[" + component + "]", msg);
```

Comment:
This is also not where you should be calling getFPGASeconds. I said to do it in the pushLogs loop.